susemanager-utils/testing/docker/head/suma-head-nodejs/add_repositories.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+zypper ar -f http://download.suse.de/ibs/Devel:/Galaxy:/nodejs/SLE_12_SP3/ "nodejs"
```

Comment:
If you add the repo here, you do not need to add it to the base image.

Reply:
Good catch @mcalmer. Removed it from base image.